test/e2e/app-dir/app/index.test.ts

```diff
@@ -169,32 +168,32 @@ describe('app dir - basic', () => {
     })
   }
 
-  // Turbopack has different chunking in dev/production which results in the entrypoint name not being included in the outputs.
-  if (!process.env.IS_TURBOPACK_TEST) {
-    it('should encode chunk path correctly', async () => {
-      await next.fetch('/dynamic-client/first/second')
-      const browser = await next.browser('/')
-      const requests = []
-      browser.on('request', (req) => {
-        requests.push(req.url())
-      })
+  it('should encode chunk path correctly', async () => {
+    const requests = []
+    const browser = await next.browser('/dynamic-client/first/second', {
+      beforePageLoad(page) {
+        page.on('request', (req) => {
+          requests.push(req.url())
+        })
+      },
+    })
 
-      await browser.eval(
-        'window.location.href = "/dynamic-client/first/second"'
-      )
+    await browser.waitForElementByCss('#id-page-params')
 
-      await browser.waitForElementByCss('#id-page-params')
+    expect(requests).not.toEqual(
+      expect.arrayContaining([expect.stringContaining('[category]')])
+    )
 
-      expect(
-        requests.some(
-          (req) =>
-            req.includes(
-              encodeURI(isTurbopack ? '[category]_[id]' : '/[category]/[id]')
-            ) && req.includes('.js')
-        )
-      ).toBe(true)
-    })
-  }
+    // Turbopack doesn't recreate the original folder structure for the output chunks
+    if (!process.env.IS_TURBOPACK_TEST) {
```

Comment:
Should we prefer using `isTurbopack` from `nextTestSetup` here?

Reply:
it's used very inconsistently 🤷